src/coffee/directives/api/models/child/window-child-model.coffee

```diff
@@ -54,44 +58,46 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
 
         createGWin: (isOpen = false) =>
           maybeMarker = @getGmarker()
-          unless @gObject?
-            defaults = {}
-            if @opts?
-              #being double careful for race condition on @opts.position via watch coords (if element and coords change at same time)
-              @opts.position = @getCoords @scope.coords if @scope.coords
-              defaults = @opts
-            if @element
-              @html = if _.isObject(@element) then @element.html() else @element
-            _opts = if @scope.options then @scope.options else defaults
-            @opts = @createWindowOptions maybeMarker, @markerScope or @scope, @html, _opts
-
-          if @opts? and !@gObject
-            if @opts.boxClass and (window.InfoBox and typeof window.InfoBox is 'function')
-              @gObject = new window.InfoBox @opts
-            else
-              @gObject = new google.maps.InfoWindow @opts
-            @handleClick(@scope?.options?.forceClick or isOpen)
-            @doShow()
-
-            # Set visibility of marker back to what it was before opening the window
-            @listeners.push google.maps.event.addListener @gObject, 'domready', ->
-              ChromeFixes.maybeRepaint @content
-
-            @listeners.push google.maps.event.addListener @gObject, 'closeclick', =>
-              if maybeMarker
-                maybeMarker.setAnimation @oldMarkerAnimation
-                if @markerIsVisibleAfterWindowClose
-                  _.delay => #appears to help animation chrome bug
-                    maybeMarker.setVisible false
-                    maybeMarker.setVisible @markerIsVisibleAfterWindowClose
-                  , 250
-              @gObject.close()
-              @model.show = false
-              if @scope.closeClick?
-                @scope.$evalAsync @scope.closeClick()
+
+          defaults = {}
```

Comment:
basically moved stuff around so createGWin can play with an existing gObject and set its content without having to erase it.